src/weatherSource/WeatherSource.js

```diff
@@ -69,7 +69,7 @@ class OpenWeatherMapApi {
             data.coord.lat, 
             data.coord.lon, 
             data.name, 
-            data.sys.country ? data.sys.country : "None"
+            data.sys.country ? data.sys.country : undefined
```

Comment:
O ile się orientuję to w poprzednim przypadku kiedy mieliśmy "None", JS pozwala wygodnie zrobić to przez symbol `||`, zamiast `variable ? variable : someOtherValue` można po prostu napisać `variable || someOtherValue`. Wracając jednak do tej zmiany, to sprawdzenie czy zmienna jest `undefined` a jeśli tak, to przypisanie jej `undefined` wydaje się trochę zbędne 😄 wystarczyłoby samo `data.sys.country`

Reply:
kurde, ale gafa xD. Dzisiaj słabo mi idzie, zwalam to na ból głowy :S